### ISSUE TITLE
Please provide demo apk in gplay

### ISSUE THREAD

**Issue by Radzhab** [davideas/FlexibleAdapter#3]
Thx

**Comment by davideas** [comment]
It's on the to do list, but unfortunately it will *not* be soon.

**Comment by Rainer-Lang** [comment]
+1 for me

**Comment by Rainer-Lang** [comment]
Compile sample-app works fine.

**Comment by davideas** [comment]
To try the demo, you can now download the apk from the latest [release](https://github.com/davideas/FlexibleAdapter/releases).
I've not created account on G.play yet :-(

**Comment by danielpassos** [comment]
@davideas I can help on it if you want.

**Comment by bfayette** [comment]
The app demo is impressive, at least from a user point of view.  Unfortunately, I came accross this library after a lengthy search on how to use effectively recyclerview/cardview to show data. It should definitively have a better referencement. All the basic functionnalities are there.

**Comment by krupalagile** [comment]
Yes, demo is good, Screenshot are not much good. Please make demo live on playstore.

**Comment by droidluv** [comment]
The current downloadable pre-built rc3 apk in https://github.com/davideas/FlexibleAdapter/releases is just downright crashing(Mi A1 - Android One device- vanilla Android) or not installing in some devices (Redmi 4A - MIUI 8)

**Comment by davideas** [comment]
@droidluv, I've provided a new apk of the Snapshot of today, I hope it will work on those devices.
About play store, I want to publish it as final release and the time is the few I have, giving priority to the library improvements and fixes.
